src/pages/transaction-signing/components/event-card.tsx

```diff
@@ -1,34 +1,35 @@
 import React from 'react';
-import { Box, color, IconButton, Stack, Text } from '@stacks/ui';
-import { SpaceBetween } from '@components/space-between';
-import { FiMoreHorizontal as IconDots } from 'react-icons/fi';
-import { Caption } from '@components/typography';
-import { AssetItem } from '@pages/transaction-signing/components/asset-item';
+import {Box, color, IconButton, Stack, Text} from '@stacks/ui';
+import {SpaceBetween} from '@components/space-between';
+import {FiMoreHorizontal as IconDots} from 'react-icons/fi';
+import {Caption} from '@components/typography';
+import {AssetItem} from '@pages/transaction-signing/components/asset-item';
+import {SendFormSelectors} from "@tests/page-objects/send-form.selectors";
 
 export const TransactionEventCard: React.FC<any> = ({
-  title,
-  left,
-  right,
-  amount,
-  ticker,
-  icon,
-  message,
-  actions,
-  isLast,
-}) => {
+                                                      title,
+                                                      left,
+                                                      right,
+                                                      amount,
+                                                      ticker,
+                                                      icon,
+                                                      message,
+                                                      actions,
+                                                      isLast,
+                                                    }) => {
   return (
     <>
       <Stack spacing="base-loose" p="base-loose">
         <SpaceBetween position="relative">
           <Text fontWeight={500} fontSize={2}>
-            {title}
+            <span data-testid={SendFormSelectors.TransferMessage}>{title}</span>
           </Text>
-          {actions && <IconButton size="24px" icon={IconDots} position="absolute" right={0} />}
+          {actions && <IconButton size="24px" icon={IconDots} position="absolute" right={0}/>}
```

Comment:
More formatter issues. Try and configure your IDE properly, lmk if you need help.